### DISCUSSION TITLE
Is there a generic way of getting a reference to flets page object?

### DISCUSSION THREAD

**Discussion by ItsCubeTime** [flet-dev/flet#787] [Q&A]
Hi 👋 Is  there some function callback that I can use to get a page ref in parts of my code that isnt directly called by some part of flet?

If not, make this a feature request. I think this is the most obnoxious "issue" Ive run into so far

**Comment by FeodorFitsner** [comment]
Flet could run multiple apps in the same Python app. I'm not sure there is an easy way to do "get current page" method.

Recommended solution is to implement your app as a `UserControl`.
